src/components/markdown-renderer/hooks/use-reveal.ts

```diff
@@ -38,25 +38,32 @@ export const useReveal = (markdownContentLines: string[], slideOptions?: SlideOp
     }
     setRevealStatus(REVEAL_STATUS.INITIALISING)
     log.debug('Initialize with slide options', slideOptions)
-    const reveal = new Reveal({})
-    reveal
-      .initialize()
-      .then(() => {
-        reveal.layout()
-        reveal.slide(0, 0, 0)
-        reveal.addEventListener('slidechanged', (event) => {
-          currentSlideState.current = {
-            indexHorizontal: event.indexh,
-            indexVertical: event.indexv ?? 0
-          } as SlideState
-        })
 
-        setDeck(reveal)
-        setRevealStatus(REVEAL_STATUS.INITIALISED)
-        log.debug('Initialisation finished')
+    import('reveal.js')
```

Comment:
I guess we don't need to put this in it's own chunk, because it will be put in it's own chunk? Maybe we want to make that explicit anyway?

Reply:
The chunk names are generated automatically. The additional name is just for the developer.

Reply:
Okay, but don't we want to make it explicit instead of implicit?